### PR TITLE
fix(promise): fullfill -> fulfill

### PR DESCRIPTION
**Pull request by standbyoneself**
### Description

There is a grammar error fix

### Motivation

Now it's very inconsistent to read same articles about Promise with correct "fulfillment" and incorrect "fullfillment"

### CODE REVIEW

**Review by cw118 — APPROVED** [review]
LGTM for l10n-fr. Thank you :)

File: files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ p.then(function(value) {
 ### 參數
 
 - `onFulfilled`
-  - : 一個 {{jsxref("Function")}}，當 `Promise` 被實現（fulfilled）時被呼叫。此函式接收一個實現值（`fullfillment value）作為引數。`
+  - : 一個 {{jsxref("Function")}}，當 `Promise` 被實現（fulfilled）時被呼叫。此函式接收一個實現值（`fulfillment value）作為引數。`
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md:27]:
```suggestion
  - : 一個 {{jsxref("Function")}}，當 `Promise` 被實現（fulfilled）時被呼叫。此函式接收一個實現值（`fulfillment value`）作為引數。
```

File: files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ p.then(function(value) {
...
 - `onRejected` {{optional_inline}}
   - : 一個 {{jsxref("Function")}}，當 `Promise` 被拒絕（rejected）時被呼叫。此函式接收一個失敗訊息（`rejection reason）作為引數。`
```
Comment (yin1999) [inline files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md:29]:
```suggestion
  - : 一個 {{jsxref("Function")}}，當 `Promise` 被拒絕（rejected）時被呼叫。此函式接收一個失敗訊息（`rejection reason`）作為引數。
```

**Review by yin1999 — APPROVED** [review]
LGTM for l10n-zh, two suggestions on bad code fence position

**Review by josielrocha — APPROVED** [review]
LGTM pt-br

**Review by SphinxKnight — APPROVED** [review]
sign-off for @mdn/yari-content-fr

File: files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md (hunk 1 above)
Comment (standbyoneself) [reply files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md]: Nice catch. Fixed it

File: files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md (hunk 2 above)
Comment (standbyoneself) [reply files/zh-tw/web/javascript/reference/global_objects/promise/then/index.md]: Fixed it!

**Review by sashasushko — APPROVED** [review]

**Review by cos18 — APPROVED** [review]
LGTM in ko!

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja` locale.

**Comment by cw118** [comment]
It seems that all locales have approved, let's merge this. Thank you @standbyoneself :)

**Pull request merged**